README.md

```diff
@@ -80,18 +80,20 @@ colcon build
 
 ## Run
 
-The `TURTLEBOT3_MODEL` environment variable must be set when running the simulation application (not needed for robot application). Valid values are `burger`, `waffle`, and `waffle_pi`.
+The `TURTLEBOT3_MODEL` environment variable must be set when running both robot and simulation application. Valid values are `burger`, `waffle`, and `waffle_pi`.
```

Comment:
Does this one have a default value?

Reply:
After some discussion, there is no avoiding the need to set the `TURTLEBOT3_MODEL` environment variable, because it is not optional for https://github.com/ROBOTIS-GIT/turtlebot3/blob/master/turtlebot3_navigation/launch/move_base.launch.